src/components/TodoTask/TodoTask.tsx

```diff
@@ -0,0 +1,143 @@
+import React, {FC, useEffect, useRef, useState} from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { ErrorType } from '../../types/Error';
+
+type Props = {
+  todo: Todo;
+  isLoading: boolean;
+  error: ErrorType;
+  removeTodo: (todoId: number) => void;
+  changeTodo: (todoId: number, updatedData: Partial<Todo>) => void;
+};
+
+export const TodoTask: FC<Props> = ({
+  todo,
+  isLoading,
+  removeTodo,
+  changeTodo,
+  error,
+}) => {
+  const [isEditing, setEdited] = useState(false);
```

Comment:
Consider renaming 'setEdited' to 'setIsEditing' for consistency with the state variable name 'isEditing'.